content/docs/guides/query-log.md

```diff
@@ -39,7 +39,7 @@ scrape_configs:
   - targets: ['localhost:9090']
 ```
 
-Then, [reload](../prometheus/latest/management_api/#reload) the Prometheus configuration:
+Then, [reload](./prometheus/latest/management_api/#reload) the Prometheus configuration:
```

Comment:
My comment was forgotten. I don't think this will work either. Laura use a link with an absolute path `/docs/prometheus/...`.